pytorch/inference/docker/1.11/py3/Dockerfile.cpu

```diff
@@ -122,8 +121,13 @@ RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases
     requests \
     # Below 2 are included in miniconda base, but not mamba so need to install
     conda-content-trust \
-    charset-normalizer \
- && /opt/conda/bin/conda clean -ya
+    charset-normalizer
+# Upstream conda looks to have moved to 4.13 which is incompatible with mamba 0.22.1 and will fail the conda-forge installs.
+# having "conda update conda" before the "conda -c conda-forge" commands will automatically update conda to 4.13.
+# Moving conda update conda" after the "conda -c conda-forge" commands keep conda at 4.12 but will update other packages using
+# the current conda 4.12
+RUN /opt/conda/bin/conda update -y conda \
```

Comment:
Why did you add another RUN to create an extra layer, and make the following `conda clean` less effective?

Reply:
Upstream conda looks to have moved to 4.13 which is incompatible with mamba 0.22.1 and will fail the conda-forge installs.Having "conda update conda" before the "conda -c conda-forge" commands will automatically update conda to 4.13.Moving conda update conda" after the "conda -c conda-forge" commands keep conda at 4.12 but will update other packages using the current conda 4.12.